src/screens/landingPageScreens/ApplicationInfoScreen.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react";
+import BlankImg from "/images/Landing Page Images/BlankImage.png";
+
+interface CardData {
+    title: string;
+    description: string;
+}
+
+interface CardProps {
+    title: string;
+    description: string;
+    isOpen: boolean;
+    onClick: () => void;
+}
+
+const CardList = () => {
+    const [openCardIndex, setOpenCardIndex] = useState<number | null>(null);
+
+    const cardsData: CardData[] = [
```

Comment:
This cound be moved to a dedicated file for static data, e.g., `src/data/CardData`, and renamed to something like `applicationInfoCards: CardData[]`.